Gems/Atom/RPI/Code/Source/RPI.Public/Scene.cpp

```diff
@@ -418,12 +483,139 @@ namespace AZ
             }
         }
 
+        void Scene::CollectDrawPacketsTaskGraph()
+        {
+                AZ_PROFILE_SCOPE(RPI, "CollectDrawPackets");
+                AZ::TaskGraphEvent collectDrawPacketsTGEvent;
+                static const AZ::TaskDescriptor collectDrawPacketsTGDesc{"RPI_Scene_PrepareRender_CollectDrawPackets", "Graphics"};
+
+                AZ::TaskGraph collectDrawPacketsTG;
+                // Launch FeatureProcessor::Render() jobs
+                for (auto& fp : m_featureProcessors)
+                {
+                    collectDrawPacketsTG.AddTask( 
+                        collectDrawPacketsTGDesc,
+                        [this, &fp]()
+                        {
+                            fp->Render(m_renderPacket);
+                        });
+
+                }
+                collectDrawPacketsTG.Submit(&collectDrawPacketsTGEvent);
+
+                // Launch CullingSystem::ProcessCullables() jobs (will run concurrently with FeatureProcessor::Render() jobs if m_parallelOctreeTraversal)
+                bool parallelOctreeTraversal = m_cullingScene->GetDebugContext().m_parallelOctreeTraversal;
+                m_cullingScene->BeginCulling(m_renderPacket.m_views);
+                AZ::JobCompletion processCullablesCompletion;
+                for (ViewPtr& viewPtr : m_renderPacket.m_views)
+                {
+                    AZ::Job* processCullablesJob = AZ::CreateJobFunction([this, &viewPtr](AZ::Job& thisJob)
```

Comment:
Could these jobs roll into the same TG above?

Reply:
I do want to do that in the future. When I looked at those jobs they are more complex than the other converted jobs and spawn child jobs they wait on as well.